pulp-ostree.spec

```diff
@@ -69,6 +69,7 @@ rm -rf %{buildroot}
 %package -n python-pulp-ostree-common
 Summary: Pulp OSTree support common library
 Group: Development/Languages
+Provides: python2-pulp-ostree-common
```

Comment:
Due to fedora naming conventions, the packages that are in fedora that are named python-* are named python2-*.  These fedora packages have a 'Provides:' line for python-*.  Since we don't provide a python2-* from upstream, these packages conflict with each other.

This only exists for packages prefixed with python, as this denotes a python library.  Other packages are not prefixed with python, so proper package hierarchy exists.